src/search/landmarks/landmark_cost_assignment.h

```diff
@@ -11,31 +13,35 @@ class OperatorsProxy;
 namespace landmarks {
 class LandmarkGraph;
 class LandmarkNode;
+class LandmarkStatusManager;
 
 class LandmarkCostAssignment {
     const std::set<int> empty;
 protected:
+    LandmarkStatusManager &lm_status_manager;
     const LandmarkGraph &lm_graph;
     const std::vector<int> operator_costs;
 
     const std::set<int> &get_achievers(int lmn_status,
                                        const LandmarkNode &lmn) const;
 public:
     LandmarkCostAssignment(const std::vector<int> &operator_costs,
-                           const LandmarkGraph &graph);
+                           const LandmarkGraph &graph,
+                           LandmarkStatusManager &lm_status_manager);
     virtual ~LandmarkCostAssignment() = default;
 
-    virtual double cost_sharing_h_value() = 0;
+    virtual double cost_sharing_h_value(const GlobalState &state) = 0;
 };
 
 class LandmarkUniformSharedCostAssignment : public LandmarkCostAssignment {
     bool use_action_landmarks;
 public:
     LandmarkUniformSharedCostAssignment(const std::vector<int> &operator_costs,
```

Comment:
What's the problem here?

Reply:
If we need to break lines here, I think we prefer to make line break after the opening parenthesis so that the next line starts with `const std::vector<int>` on a one-level indentation compared to the constructor. Then all further parameters are not as extremely indented as they are now.